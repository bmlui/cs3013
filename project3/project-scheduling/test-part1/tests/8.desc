SJF: properly handle idle CPUs for multiple times
