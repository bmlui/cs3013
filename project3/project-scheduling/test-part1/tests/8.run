./scheduler 0 SJF tests/8.in 0
