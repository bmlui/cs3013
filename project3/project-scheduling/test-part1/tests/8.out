Execution trace with SJF:
t=0: [Job 0] arrived at [0], ran for: [1]
t=2: [Job 1] arrived at [2], ran for: [5]
t=7: [Job 4] arrived at [7], ran for: [1]
t=8: [Job 3] arrived at [5], ran for: [4]
t=12: [Job 2] arrived at [3], ran for: [11]
t=25: [Job 5] arrived at [25], ran for: [4]
End of execution with SJF.
